mercury.spec

```
@@ -26,18 +26,8 @@ Group:    Development/Libraries
 License:  Argonne National Laboratory, Department of Energy License
 URL:      http://mercury-hpc.github.io/documentation/
 Source0:  https://github.com/mercury-hpc/mercury/archive/v%{dl_version}.tar.gz
-Patch0:   https://github.com/daos-stack/mercury/cpu_usage.patch
-Patch1:   https://github.com/daos-stack/mercury/cxi_provider_plus_daos_9561.patch
-Patch2:   https://github.com/daos-stack/mercury/na_ucx_changes.patch
 
-%if 0%{?suse_version} > 0
-BuildRequires:  libatomic1
-%else
-%if 0%{?rhel} < 8
-BuildRequires:  openpa-devel
-%endif
-%endif
-BuildRequires:  libfabric-devel >= 1.9.0-5
+BuildRequires:  libfabric-devel = 1.14.0
```

We **really** want to avoid this kind of single `version-release` pinning whenever possible.  Ideally this should be `>=`.

```suggestion
BuildRequires:  libfabric-devel >= 1.14.0
```